spec/unit/vagrant-proxyconf/action/configure_docker_proxy_spec.rb

```diff
@@ -1,10 +1,500 @@
 require 'spec_helper'
+require 'vagrant-proxyconf/config/proxy'
 require 'vagrant-proxyconf/action/configure_docker_proxy'
+require 'pry'
```

Comment:
If `pry` is needed, it should be added as direct dependency to the `Gemfile`.

Reply:
It's not needed, just helpful in debugging and it's installed it seems to be installed as a development dependency just not sure which one.  I'll remove it from the spec files.

Reply:
Resolved via: d951ca8